script/releasing/releaser.py

```diff
@@ -149,22 +303,140 @@ def release_version(NEXT_RELEASE):
+        action='store_true',
+        help="Publish a new version of EVA.",
+    )
+
+    parser.add_argument(
+        "-p",
+        "--publish-to-pypi",
+        action='store_true',
+        help="Create wheels and upload to PyPI.",
+    )
+
+    parser.add_argument(
+        "-a",
+        "--upload-assets",
+        action='store_true',
+        help="Upload assets on Github.",
+    )
+
+    parser.add_argument(
+        "-b",
+        "--bump-up-version",
+        action='store_true',
+        help="Bump up version.",
     )
 
     args = parser.parse_args()
 
+    # FIGURE OUT NEXT VERSION
+
+    current_version = VERSION
+    version = Version(current_version)
+    print("CURRENT VERSION: " + current_version)
+
+    selected_release_type = args.next_release_type
+
+    # Get the string representation of the next version
+    if selected_release_type == SemanticVersionTypeEnum.PRERELEASE:
+        next_version = version.next_patch()
+        next_version_str = "v" + str(next_version) + ".alpha"
+    elif selected_release_type == SemanticVersionTypeEnum.PATCH:
+        next_version = version.next_patch()
+        next_version_str = "v" + str(next_version)
+    elif selected_release_type == SemanticVersionTypeEnum.MINOR:
+        next_version = version.next_minor()
+        next_version_str = "v" + str(next_version)
+    elif selected_release_type == SemanticVersionTypeEnum.MAJOR:
+        next_version = version.next_major()    
+        next_version_str = "v" + str(next_version)
+
+    current_version_str_without_dev = str(version)
+    print("CURRENT VERSION WITHOUT DEV: " + current_version_str_without_dev)
+    print("NEXT    VERSION : " + next_version_str)
+
     if args.get_changelog:
-        get_changelog(args.get_changelog)
+        release_date = get_commit_id_of_latest_release()
+        changelog = get_changelog(release_date)
+        print(changelog)
+
+    if args.update_changelog:
+        release_date = get_commit_id_of_latest_release()
+        changelog = get_changelog(release_date)
+        append_changelog(changelog, current_version_str_without_dev)
 
     if args.release_version:
-        release_version(args.release_version)
+        release_version(current_version_str_without_dev)
+
+    if args.publish_to_pypi:
+        publish_wheels(current_version_str_without_dev)
+
+    if args.upload_assets:
+        release_date = get_commit_id_of_latest_release()
+        changelog = get_changelog(release_date)
+        upload_assets(changelog, current_version_str_without_dev)
+
+    if args.bump_up_version:
+        bump_up_version(next_version)
+
+    ## DO EVERYTHING BY DEFAULT
+
+    # GET CHANGELOG 
+    release_date = get_commit_id_of_latest_release()
+    changelog = get_changelog(release_date)
+    print(changelog)
+
+    # UPDATE CHANGELOG
+    append_changelog(changelog, current_version_str_without_dev)
+
+    # RELEASE VERSION
+    release_version(current_version_str_without_dev)
+
+    # PUBLISH WHEELS ON PYPI
```

Comment:
We should not automate this to avoid accidentally uploading to PyPi (cannot be reverted)